modules/oe_webtools_page_feedback/src/Form/WebtoolsPageFeedbackSettingsForm.php

```diff
@@ -0,0 +1,63 @@
+<?php
+
+declare(strict_types = 1);
+
+namespace Drupal\oe_webtools_page_feedback\Form;
+
+use Drupal\Core\Form\ConfigFormBase;
+use Drupal\Core\Form\FormStateInterface;
+
+/**
+ * Provides configuration form for the Page Feedback Form webtools widget.
+ */
+class WebtoolsPageFeedbackSettingsForm extends ConfigFormBase {
+
+  /**
+   * {@inheritdoc}
+   */
+  public function getFormId(): string {
+    return 'oe_webtools_page_feedback_settings';
+  }
+
+  /**
+   * {@inheritdoc}
+   */
+  public function buildForm(array $form, FormStateInterface $form_state): array {
+    $config = $this->config('oe_webtools_page_feedback.settings');
+
+    $form['enabled'] = [
+      '#type' => 'checkbox',
+      '#title' => $this->t('Enabled'),
+      '#description' => $this->t('Check this box if you would like to enable the Page feedback form on all the content pages.'),
+      '#default_value' => $config->get('enabled'),
+    ];
+    $form['feedback_form_id'] = [
+      '#type' => 'number',
+      '#title' => $this->t('Form ID'),
+      '#description' => $this->t('Provide your webtools form ID.'),
+      '#default_value' => $config->get('feedback_form_id'),
+      '#required' => TRUE,
```

Comment:
Hmm, this should only be required if the thing is enabled right? So if they first enable and set a form ID, then they need to leave the form ID there if they wanna disable. So let's add a #state here for that.